service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunnerTest.java

```diff
@@ -49,8 +49,11 @@ public class AzureDatabaseUtilsRunnerTest extends BaseMockitoStrictStubbingTest
   @Mock private ApiAzureLandingZoneDeployedResource mockDatabase;
   @Mock private ApiAzureLandingZoneDeployedResource mockAdminIdentity;
 
+  private final String AZURE_ENV = "AzureCloud";
+
   @BeforeEach
   public void createAzureDatabaseUtilsRunner() {
+    when(mockAzureConfig.getAzureEnvironmentConfigString()).thenReturn(AZURE_ENV);
```

Comment:
in practice, we should have a value for the `AZURE_ENVIRONMENT` env var. This test was using an empty mock for its config. Now it has a value.